Aspire Budgeting/AppDelegate.swift

```diff
@@ -9,6 +9,11 @@ import UIKit
 class AppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/2d329511ce736e743f4863ca4c87513f4fad81ea/Aspire Budgeting/AppDelegate.swift#L9"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>